internal/pkg/source/source.go

```diff
@@ -232,25 +233,49 @@ func sourcesFromBlocks(fn *ssa.Function, conf classifier) []*Source {
 
 		for _, instr := range b.Instrs {
 			switch v := instr.(type) {
```

Comment:
The diff is really weird, I recommend looking at the code before/after in an editor.